user-journey.adoc

```diff
@@ -3,20 +3,26 @@
  *Authors:* Lincoln Baxter, III and Charles Moulliard, Diógenes Rettori
  *Version:* 0.6 - April 25, 2018
 
-This document describes the steps required to install a local Istio Openshift instance, necessary prerequisites (JDK, Maven, Launcher, Jaeger, …), and the operations to grant access for the user to the project. It also describes how the user can experiment with a mission selected using the launcher
-
-The Fabric8 Launcher provides two workflows to play with an Istio mission (zip vs. git flow), and this document will apply to either - described below:
+This document describes the steps required to install a local Istio Openshift instance, necessary prerequisites, and describes how the your can experiment with Istio enabled missions using the launcher.
 
 == Flows description
```

Comment:
This description of the flows is much clearer than the previous version.

Reply:
thanks :)